src/ConfigCatClient.ts

```diff
@@ -55,12 +55,20 @@ export interface IConfigCatClient {
 
     /** Returns the values of all feature flags or settings */
     getAllValuesAsync(user?: User): Promise<SettingKeyValue[]>;
+
+    /** Sets the default user for feature flag evaluations. 
+     * In case the getValue function isn't called with a UserObject, this default user will be used instead. */
+    setDefaultUser(defaultUser?: User): void;
```

Comment:
nullable parameter and `clearDefaultUser()` look redundant to me, can you explain why it is necessary (nullable parameter)?

Reply:
I think it gives more flexibility in using this feature.

Reply:
Yes, plus a nullable parameter needs to be explained, `clearDefaultUser()` is self-explanatory.

Reply:
I updated the setDefaultUser to non nullable

Reply:
done